Tutorials/Level1Tutorial.md

```diff
@@ -0,0 +1,793 @@
+The SST-Elements package provides individual *components* that simulate specific 
+pieces of hardware.  The SST-Elements package provides a number of sample
+hardware elements such as memories, caches, processor cores and network interfaces.  
+It also provides a number of examples on how to construct larger simulations of 
+full systems.  The SST-Elements provides a number of components that are built 
+by default as well as a number of components that require external libraries.  
+For the purposes of this tutorial, we will build the SST-Elements package 
+with the default options.
+
+Much in the same manner as the SST-Core package, we can select the latest release 
+package of SST-Elements or build directly from the git source tree.  
+The standard SST-Elements release packages are available on the 
+[release page](http://sst-simulator.org/SSTPages/SSTMainDownloads/)
+or check out the source code from [Github](https://github.com/sstsimulator/sst-elements).
+
+If you download the release package, untar it as follows:
+```
+$> tar xzvf sstelements-X.Y.Z.tar.gz
+$> cd sstelements-X.Y.Z
+```
+
+If you seek to clone the source repository directly, do the following.  Note 
+the final autogen command to generate the required configuration scripts.  This 
+is only required if you clone the source repository directly.
+```
+$> git clone https://github.com/sstsimulator/sst-elements.git sst-elements
+$> cd sst-elements
+$> ./autogen.sh
+```
+
+Now that you have the SST-Elements source, you should select an appropriate 
+installation location.  For this example, we will utilize `/opt/sst/elements` as our 
+installation target.  At this point, we can configure, build and install 
+the SST-Elements infrastructure as follows:
+```
+$> mkdir /opt/sst/elements
+$> ./configure --prefix=/opt/sst/elements --with-sst-core=/opt/sst
+$> make -j
+$> make install
+$> sst-info -q
+```
+
+At this point in the tutorial, we have a functional SST environment 
+with both the base SST-Core and the SST-Elements packages installed.
+
+## Basic Commands <a name="BasicCommands"></a>
+
+There are a number of basic commands that are commonly utilized when 
+running and/or debugging an SST simulation.  We summarize the most 
+commonly used command sets here and the situations where they are 
+appropriately utilized.  However, all of the major SST command line tools 
+provide a `--help` option to print useful command line options 
+and their associated parameters.  
+
+### SST Execution and Utility Functions
+
+* _Executing a Simulation_: The most common command line tool is the `sst` command.  
+This is utilized to build and execute a simulation run using the SST Core and associated 
+Elements libraries.  Using the `sst` command line tool and passing a loadable 
+simulation graph using Python or JSON can be performed as follows:
+```
+$> sst /path/to/simulation.py
+$> sst /path/to/simulation.json
+```
+
```

Comment:
The other common basic parameter is the `--model-options="args to pass to simulation.py"` or equivalent syntax `sst /path/to/simulation.py -- <args to pass to simulation.py>`